charts/datadog/templates/NOTES.txt

```diff
@@ -211,19 +211,47 @@ To enable it please set clusterAgent.enabled to 'true'.
 {{- end }}
 
 {{- if .Values.providers.gke.autopilot}}
+
+###########################################################################################
+####   WARNING: Only one Datadog chart release allowed by namespace on GKE Autopilot   ####
+###########################################################################################
+
+{{- if not .Values.agents.rbac.create }}
+
+########################################################################
+####   WARNING: agents.rbac.create must be true on GKE Autopilot   ####
+########################################################################
+
+{{- fail "On GKE Autopilot environments, the option 'agents.rbac.create' must be set to 'true'" }}
+
+{{- end }}
+
 {{- if eq (include "system-probe-feature" .) "true" }}
 
-##################################################################################
-####               WARNING: System Probe is not supported on GKE Autopilot    ####
-##################################################################################
+#####################################################################
+####   WARNING: System Probe is not supported on GKE Autopilot   ####
+#####################################################################
```

Comment:
Is there a specific reason for which:
* If we try to enable the CWS or CSPM on GKE autopilot, it fails whereas
* If we try to enable a `system-probe` feature, it is only a warning.

Reply:
it is because, the feature after is disabled silently. I can also put a fail here.